source/testing/testing-components.md

```diff
@@ -201,6 +201,41 @@ test('trigger external action when button is clicked', function(assert) {
 });
 ```
 
+### Testing component blocks and defaults
+
+Often you might define a component which works in block form:
+
+```app/templates/components/my-component.hbs
+{{#if template}}
```

Comment:
We intend to add `hasBlock?` helper for this, I am unsure that we should suggest this (though checking `template` will likely always work).